### PR TITLE
SGV Enterprise: The Aircraft Carrier

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

Introducing...Enterprise!
A SolGov carrier on loan to Nanotrasen forces for the duration of the Syndicate war, the Enterprise is equipped with state of the art systems and can deploy up to a theoretical 100 fighters at once. It features a fully dedicated runway deck, advanced chemistry and medical facilities. While this ship is impressive, it lacks heavy armament of its own, and relies on its fighter complement to deal damage.

This map has been designed with a focus on streamlining hallways and allowing for an ungodly amount of fighters to be deployed. Many thanks to Goat Monarch for their ongoing help with mapping and helping me plan this!

Deck 1 (Flight deck)

Deck 2 (Operations deck)

Bugs resolved:
fixes https://github.com/BeeStation/NSV13/issues/173
fixes https://github.com/BeeStation/NSV13/issues/142
fixes https://github.com/BeeStation/NSV13/issues/279

## Changelog
:cl: Kmc2000 with GoatMonarch (Special thanks to Cenrus and Sergei)
add: Added SGC Enterprise, a 2-deck carrier ship
/:cl:

### CODE REVIEW

File: config/maps.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ Format:
 	disabled (disables the map)
 endmap
 
-map hammerhead
+map enterprise
```
Comment (Vasily2013) [inline config/maps.txt]: Do not remove hammerhead from the maps.txt

File: nsv13.dme [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\enterprise.dm"
```
Comment (Vasily2013) [inline nsv13.dme]: DON'T. CHANGE. THE. DME.

File: nsv13/code/game/objects/structures/barricade.dm [hunk 3]
```diff
@@ -58,6 +58,12 @@
 	anchored = FALSE
 	build_state = 0
 
+/obj/structure/peacekeeper_barricade/metal/plasteel/deployable/active
```
Comment (Vasily2013) [inline nsv13/code/game/objects/structures/barricade.dm]: This belongs in a different PR

File: nsv13/code/modules/turbolift/turbolift_changes.dm [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/obj/machinery/door/airlock/turbolift/ship
```
Comment (Vasily2013) [inline nsv13/code/modules/turbolift/turbolift_changes.dm]: the name of this file suggests that it is intended as a change to another file.

File: nsv13/game/custom_items.dm [hunk 5]
```diff
@@ -130,4 +130,13 @@
 	. = ..()
 	if(start_turf)
 		start_turf = null
-		to_chat(user, "<span class='warning'>Cleared target selection</span>")
\ No newline at end of file
+		to_chat(user, "<span class='warning'>Cleared target selection</span>")
+		add_fingerprint(user)
+
+/obj/item/reagent_containers/food/drinks/solgovcup //Credit to baystation for this sprite!
```
Comment (Vasily2013) [inline nsv13/game/custom_items.dm]: This should be in a different PR

File: nsv13/code/game/objects/structures/large_stairs.dm [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+* Stairs that behave like stairs actually should because /tg/ stairs are a pile of ass.
+* Just place stairs facing up and down above each other.
+* These stairs only go up one level. If you stack a tonne of them you won't have a good time
+* Make sure theyre facing the same direction or it'll feel weird traversing them.
+* MAPPERS: Please enclose your stairs with walls. Otherwise theyll feel weird.
+*
+*
+*/
+
+/obj/structure/stairs/large_stairs
```
Comment (Vasily2013) [inline nsv13/code/game/objects/structures/large_stairs.dm]: This should also be in another PR, but i'll allow it to piggyback off of Enterprise.

File: nsv13/code/modules/turbolift/turbolift_changes.dm (hunk 4 above)
Comment (Vasily2013) [reply nsv13/code/modules/turbolift/turbolift_changes.dm]: Rename this

**Comment by Cenrus** [comment]
you could fit the gateway somewhere in science, and the second deck has enough room for a teleporter.

**Comment by Cenrus** [comment]
image
 
enough room here for EVA storage, too.

**Comment by Cenrus** [comment]
image

Glass table snuck onto this tile here when I was editing the map, it's not visible on the map editor though.

**Comment by Kmc2000** [comment]
> is _maps/shuttles/turbolifts/nsv/debug_enterprise.dmm a debug map, or is that a map that should be renamed to properly indicate what it is?

Don't question it, it just works

**Comment by francinum** [comment]
Okay. No. This is not going to become a pattern of selfmerging shit.
